tests/model_registry/rbac/test_mr_rbac.py

```diff
@@ -0,0 +1,305 @@
+        )
+        with switch_context(context_to_use):
+            _, client_args = setup_mr_client(
+                model_registry_instance=model_registry_instance,
+                model_registry_namespace=model_registry_namespace,
+                admin_client=admin_client,
+            )
+
+            if use_admin_context:
+                mr_client = ModelRegistryClient(**client_args)
+                assert mr_client is not None, "Client initialization failed for admin user"
+                LOGGER.info("Successfully created Model Registry client for admin user")
+            else:
+                with pytest.raises(ForbiddenException) as exc_info:
+                    _ = ModelRegistryClient(**client_args)
+                assert exc_info.value.status == 403, f"Expected HTTP 403 Forbidden, but got {exc_info.value.status}"
+                LOGGER.info("Successfully received expected HTTP 403 status code")
+
+    @pytest.mark.sanity
+    def test_user_added_to_group(
+        self: Self,
+        model_registry_instance: ModelRegistry,
+        model_registry_namespace: str,
+        admin_client: DynamicClient,
+        test_idp_user_session: UserTestSession,
+    ):
+        """
+        Test that a user's access to Model Registry changes when added to a group.
+
+        This test verifies that:
+        1. A normal user cannot access the Model Registry initially
+        2. After adding the user to the appropriate group, they gain access
+        3. After removing the user from the group, they lose access
+
+        Args:
+            model_registry_instance: The Model Registry instance to test against
+            model_registry_namespace: The namespace where Model Registry is deployed
+            admin_client: The admin client for accessing the cluster
+            test_idp_user_session: The test user session containing both admin and user contexts
+
+        Raises:
+            AssertionError: If access permissions don't match expectations
+            ForbiddenException: Expected before group addition, unexpected after
+        """
+        LOGGER.info(
+            "-----Test that a user can access to the Model Registry once added to a "
+            "group that has the permissions to access it-----"
+        )
+        assert model_registry_instance.name == MR_INSTANCE_NAME
+        model_registry_users_group = f"{MR_INSTANCE_NAME}-users"
+
+        # Verify initial access denied
+        with switch_context(test_idp_user_session.user_context):
+            _, client_args = setup_mr_client(
+                model_registry_instance=model_registry_instance,
+                model_registry_namespace=model_registry_namespace,
+                admin_client=admin_client,
+            )
+
+            with pytest.raises(ForbiddenException) as exc_info:
+                _ = ModelRegistryClient(**client_args)
+            assert exc_info.value.status == 403
```

Comment:
This is already covered in test_user_permission(). You can remove this section.

Reply:
removed

Reply:
I still see it here.